src/constants/constants.js

```diff
@@ -11,3 +11,5 @@ export const gender = [
   { value: 'female', id: 'gender.female' },
   { value: 'other', id: 'gender.other' }
 ];
+
+export const macCheeseColor = '#efc638';
```

Comment:
I think for this you should have a different file called something like styleConstants.js, otherwise it's kinda messy.